gallery/src/components/NxGlobalHeader2/NxGlobalHeader2Page.tsx

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxTable, NxP, NxCode, NxTile, NxH2, NxTextLink, NxH3, NxList } from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile } from '../../gallery-components/GalleryTiles';
+
+import CodeExample from '../../CodeExample';
+
+const nxGlobalHeader2ExampleCode = require('./NxGlobalHeader2Example?raw'),
+    nxGlobalHeader2NoActionsExampleCode = require('./NxGlobalHeader2NoActionsExample?raw'),
+    nxGlobalHeader2DefaultLogoExampleCode = require('./NxGlobalHeader2DefaultLogoExample?raw');
+
+const NxGlobalHeader2Page = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        A page header to be used along with the <NxCode>NxGlobalSidebar2</NxCode>. So named in order to distinguish
+        it from the <NxCode>nx-page-header</NxCode> used within the legacy page layouts as well
+        as <NxCode>NxGlobalHeader</NxCode> used alongside <NxCode>NxGlobalSidebar</NxCode>.
+      </NxP>
+      <NxP>
+        Like all parts of the <NxCode>NxGlobalSidebar2</NxCode> layout, this component is only to be
+        used in full-width, section-scrolling layouts. Although the contents of <NxCode>NxGlobalHeader2</NxCode> are
+        flexible, there are strong design guidelines for the content that should be present. From left to right,
+        the header <em>should</em> contain the following items:
+      </NxP>
+      <NxList bulleted>
+        <NxList.Item>The official product logo depicting its name</NxList.Item>
+        <NxList.Item>
+          <NxList bulleted>
```

Comment:
Should this be nested under a `<NxList.Item>`? (looks a bit odd to me but doesn't really matter)
![image](https://github.com/user-attachments/assets/9891e13f-9ac9-4506-b4db-7b0fad9ac15f)

Reply:
> Should this be nested under a <NxList.Item>? (looks a bit odd to me but doesn't really matter)

Whoops yeah I didn't have that structured correctly at all. 1ff4d4ec5